### ISSUE TITLE
dojson: jobs demo records problem

### ISSUE THREAD

**Issue by jmartinm** [inspirehep/inspire-next#341]
Loading the demo records, the following error happens:

```python
Traceback (most recent call last):
  File "/Users/jmartinm/.virtualenvs/invenio-inspire/bin/inveniomanage", line 9, in <module>
    load_entry_point('invenio==2.2.0.dev20150608', 'console_scripts', 'inveniomanage')()
  File "/Users/jmartinm/.virtualenvs/invenio-inspire/src/invenio/invenio/base/manage.py", line 103, in main
    manager.run()
  File "/Users/jmartinm/.virtualenvs/invenio-inspire/lib/python2.7/site-packages/flask_script/__init__.py", line 412, in run
    result = self.handle(sys.argv[0], sys.argv[1:])
  File "/Users/jmartinm/.virtualenvs/invenio-inspire/lib/python2.7/site-packages/flask_script/__init__.py", line 383, in handle
    res = handle(*args, **config)
  File "/Users/jmartinm/.virtualenvs/invenio-inspire/src/invenio/invenio/ext/script/__init__.py", line 177, in __call__
    res = super(SignalingCommand, self).__call__(*args, **kwargs)
  File "/Users/jmartinm/.virtualenvs/invenio-inspire/lib/python2.7/site-packages/flask_script/commands.py", line 216, in __call__
    return self.run(*args, **kwargs)
  File "/Users/jmartinm/.virtualenvs/invenio-inspire/lib/python2.7/site-packages/invenio_records/manage.py", line 63, in create
    [Record.create(item) for item in data]
  File "/Users/jmartinm/.virtualenvs/invenio-inspire/src/inspire/inspire/dojson/processors.py", line 49, in convert_marcxml
    yield jobs.do(record)
  File "/Users/jmartinm/.virtualenvs/invenio-inspire/lib/python2.7/site-packages/dojson/overdo.py", line 58, in do
    output[name] = creator(output, key, value)
  File "/Users/jmartinm/.virtualenvs/invenio-inspire/src/inspire/inspire/dojson/jobs/fields/bd1xx.py", line 40, in acquisition_source
    if value.get('m'):
AttributeError: 'list' object has no attribute 'get'
```

**Comment by jmartinm** [comment]
@jalavik

**Comment by jalavik** [comment]
Fixed in 72e684c
